governance/votes/Community Executive vote - January 22, 2021.md

```diff
@@ -0,0 +1,177 @@
+
+**[Debt Ceiling](https://community-development.makerdao.com/en/learn/governance/param-debt-ceiling) Decreases**
+* The BAT-A Debt Ceiling will decrease from 10 million DAI to 2 million DAI.
+* The TUSD-A Debt Ceiling will decrease from 135 million DAI to zero DAI.
+* The Global System Debt Ceiling will decrease by 126,250,000 DAI.
+
+**[Debt Floor](https://community-development.makerdao.com/en/learn/governance/param-debt-floor) (`dust`) Changes**
+* The ETH-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The USDC-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The WBTC-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The ETH-B Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The TUSD-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The PAXUSD-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The LINK-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The COMP-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The YFI-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The UNI-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The UNIV2DAIETH-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The BAT-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The LRC-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The BAL-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The renBTC-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The AAVE-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The MANA-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The ZRX-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The KNC-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The GUSD-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The USDT-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The USDC-B Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+
+**DAI Distributions**
+* 6,300 Dai will be transferred from the Maker Protocol to @makerman at [0x9ac6a6b24bcd789fa59a175c0514f33255e1e6d0] for services rendered.
+* 3,800 Dai will be transferred from the Maker Protocol to @monet-supply at [0x8d07D225a769b7Af3A923481E1FdF49180e6A265] for services rendered.
+* 2,000 Dai will be transferred from the Maker Protocol to @Joshua_Pritikin at [0x2235A5D7bCC37855CB91dFf66334F4DFD9C39b58] for services rendered.
+* 400 Dai will be transferred from the Maker Protocol to @befitsandpiper at [0x851fB899dA7F80c211d9B8e5f231FB3BC9eca41a] for services rendered.
+* 200 Dai will be transferred from the Maker Protocol to @Vault2288 at [0x92e5a14b08e5232682eb38269a1ce661f04ec93d] for services rendered.
```

Comment:
Address is not in checksummed form. Should be 0x92e5a14b08E5232682Eb38269A1cE661F04Ec93D